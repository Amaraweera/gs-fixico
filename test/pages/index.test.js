import renderer from 'react-test-renderer';
import Dashboard from "../../src/pages/index";

describe("Dashboard Test", () => {
    it("Test Dashboard snapshot", () => {
        const component = renderer.create(
            <Dashboard/>,
          );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});