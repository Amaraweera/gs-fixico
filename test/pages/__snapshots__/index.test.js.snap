// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Test Test Dashboard snapshot 1`] = `
<h1>
  Report Your Damage...!
</h1>
`;
